services/greasyfork/greasyfork-downloads.tester.js

```diff
@@ -0,0 +1,24 @@
+import { ServiceTester } from '../tester.js'
+import { isMetric, isMetricOverTimePeriod } from '../test-validators.js'
+
+export const t = new ServiceTester({
+  id: 'GreasyForkInstalls',
+  title: 'GreasyFork Installs',
+  pathPrefix: '/greasyfork',
+})
```

Comment:
Note that once these are combined into a single class with a single export then this can be simplified to use the standard `createServiceTester` helper (more info in https://github.com/badges/shields/blob/master/doc/service-tests.md#1-boilerplate)